connectivity/FEATURE_BLE/source/cordio/source/PalAttClientImpl.cpp

```diff
@@ -392,9 +401,9 @@ void PalAttClient::on_transaction_timeout(
     }
 }
 
+#if BLE_FEATURE_GATT_CLIENT
```

Comment:
what if it's a server but not client?

Reply:
The handling is made in CordioBLE plumbing code: 

```c++
#if BLE_FEATURE_ATT
#if BLE_FEATURE_GATT_CLIENT
    AttRegister((attCback_t) impl::PalAttClient::att_client_handler);
#else
    AttRegister((attCback_t) impl::GattServer::att_cb);
#endif // BLE_FEATURE_GATT_CLIENT
#endif
```